translations.txt

```diff
@@ -595,63 +595,63 @@ Your Trial key includes all the functionality and device support of a Pro key=De
 Your Trial key requires an internet connection=Deine Testlizenz erfordert eine aktive Internetverbindung
 Your trial will expire in 15 days=Deine Testlizenz läuft in 15 Tagen ab
 Your trial will expire in 30 days=Deine Testlizenz läuft in 30 Tagen ab
-Your Unraid registration key is ineligible for replacement as it has been replaced within the last 12 months=Your Unraid registration key is ineligible for replacement as it has been replaced within the last 12 months.
+Your Unraid registration key is ineligible for replacement as it has been replaced within the last 12 months=Dein Unraid Lizenzschlüssel ist nicht zum Wechsel verfügbar da er bereits in den letzten 12 Monaten gewechselt wurde
```

Comment:
"Dein Unraid Lizenzschlüssel ist nicht zum Wechsel berechtigt da er bereits in den letzten 12 Monaten gewechselt wurde"